ISS/user_reg.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render
```

Comment:
Right now all the views for the project live in the shared `views.py` file so the contents of this should be relocated there.